### ISSUE TITLE
No lang attribute on html tag (logout state) #a11y

### ISSUE THREAD

**Issue by llune** [shaarli/Shaarli#1325]
The lang attribute on the html tag is empty on the login page.

**Comment by ArthurHoaro** [comment]
Are you sure you have a language configured in your settings? I don't reproduce your issue.

![](https://i.imgur.com/mDqcBV9.png)

**Comment by llune** [comment]
Yep French is properly set. I'll try it on a new instance because something seems indeed wrong with mine!

**Comment by nodiscc** [comment]
@llune I can't reproduce it either. Can you? If yes, can you provide:

 - platform/installation method (OS, web server, docker...)?
 - available locales on the server, and default locale (run `locale`)?
 - installed PHP extensions?
 - can you reproduce with different browsers, or a fresh [firefox profile](https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles)?

**Comment by ArthurHoaro** [comment]
Closing due to lack of feedback. Feel free to reopen if you still encounter this issue.
